target/linux/mediatek/image/filogic.mk

```diff
@@ -289,6 +289,44 @@ define Device/mercusys_mr90x-v1
 endef
 TARGET_DEVICES += mercusys_mr90x-v1
 
+define Device/jcg_q30_pro
+  DEVICE_VENDOR := JCG
+  DEVICE_MODEL := Q30 Pro
+  DEVICE_DTS := mt7981b-jcg-q30_pro
+  DEVICE_DTS_DIR := ../dts
+  DEVICE_PACKAGES := kmod-mt7981-firmware mt7981-wo-firmware
+  UBINIZE_OPTS := -E 5
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  IMAGE_SIZE := 113152k
+  KERNEL_IN_UBI := 1
+  IMAGES += factory.bin
+  IMAGE/factory.bin := append-ubi | check-size $$$$(IMAGE_SIZE)
+  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
+  KERNEL = kernel-bin | lzma | \
+	fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb
+  KERNEL_INITRAMFS = kernel-bin | lzma | \
+	fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb with-initrd
+endef
+TARGET_DEVICES += jcg_q30_pro
+
+define Device/mt7981-360-t7-108M
+  DEVICE_VENDOR := MediaTek
```

Comment:
Is this a Mediatek reference platform? DEVICE_VENDOR should math the vendor of this board.